test/DartTest.ts

```diff
@@ -306,6 +306,18 @@ describe('transpile to dart', () => {
     it('translates "void"', () => {
       chai.expect(() => translateSource('void x;')).to.throw('void operator is unsupported');
     });
+    it('translates "super()" constructor calls', () => {
+      expectTranslate('class X { constructor() { super(1); } }')
+          .to.equal(' class X { X ( ) : super ( 1 ) { /* super call moved to initializer */ ; } }');
+      expectTranslate('class X { constructor() { x(); if (y) super(1, 2); z(); } }')
+          .to.equal(' class X { X ( ) : super ( 1 , 2 ) { x ( ) ;' +
+                    ' if ( y ) /* super call moved to initializer */ ;' +
```

Comment:
Yes, I first had this only supporting super() calls as the very first child of the ctor method, then changed it to this and didn't feel to great about it.

The Traceur transpiler handles super() calls anywhere in the body but not in further children; I've  changed this to match that behaviour. We could reconsider or print a warning if it turns out to be problematic.